package.json

```diff
@@ -4,7 +4,7 @@
 	"description": "This is a Workshop about Api Testing in JavaScript",
 	"main": "index.js",
 	"scripts": {
-		"test": "mocha -t 5000"
```

Comment:
Please don't delete the tab identation, just add one tab. Do the same for nested properties as type, chai, http-status-codes and so on